apiExamples/fullscreenBreakpoint.html

```diff
@@ -0,0 +1,24 @@
+<div>
+  <auro-button id="openSmLg">Open drawer</auro-button>
+</div>
+
+<auro-drawer id="smLgDrawer" size="sm" fullscreenbreakpoint="lg">
```

Comment:
This should be `fullscreenBreakpoint`. I know it still works, but its nice for consistent documentation.